File: src/CSPAF/CSPAgents/CSPUtilityServerGroups/CSPUninstallAgent.cs
using System;
using System.Collections.Generic;

using CloudSuitePortal;
using CloudSuitePortal.Models;
using CloudSuitePortal.Services;
using CloudSuitePortal.Controllers;

namespace CSPAF.CSPAgents.CSPUtilityServerGroups
{
    class CSPUninstallAgent
    {
        public void run(Dictionary<string, object> Params)
        {
            var service = new CSPAgentService();
            service.Uninstall();
        }
    }
}

File: src/CSPAF/CSPAgents/CSPUtilityServerGroups/CSPUninstallAgent.java
package CSPAF.CSPAgents.CSPUtilityServerGroups;

import java.io.*;
import java.util.*;

import com.infor.cloudsuite.portal.singletenant.agents.services.*;

class CSPUninstallAgent
{
    public void run(HashMap<String, Object> params) throws IOException, InterruptedException, CSPAgentService.CommandException
    {
        CSPAgentService service = new CSPAgentService();
        service.uninstall();
    }
}